### PR TITLE
gimp: add page

### PR DESCRIPTION
**Pull request by marchersimon**

### CODE REVIEW

File: pages/common/gimp.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# gimp
+
+> An image manipulation and paint program.
+> More information: <https://docs.gimp.org/en/gimp-fire-up.html#gimp-concepts-running-command-line>.
+
+- Don't show the splash screen:
```
Comment (bl-ue) [inline pages/common/gimp.md:6]:
```suggestion
- Launch but don't show the splash screen:
```

File: pages/common/gimp.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# gimp
+
+> An image manipulation and paint program.
+> More information: <https://docs.gimp.org/en/gimp-fire-up.html#gimp-concepts-running-command-line>.
+
```
Comment (bl-ue) [inline pages/common/gimp.md:5]: How about giving a basic example of how to launch?
```suggestion

- Launch GIMP:

`gimp`

```

File: pages/common/gimp.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+- Don't show the splash screen:
+
+`gimp --no-splash`
+
+- Start a new instance, even if there is already a running one:
+
+`gimp --new-instance`
+
+- Open the given file as new image:
```
Comment (bl-ue) [inline pages/common/gimp.md:14]:
```suggestion
- Create and open a new image at the specified path:
```

**Review by bl-ue — COMMENTED** [review]
Nice page, @marchersimon. I've left a few comments&mdash;feel free to adjust the wording as I didn't work too hard on it 🙂

File: pages/common/gimp.md (hunk 3 above)
Comment (marchersimon) [reply pages/common/gimp.md:18]: This doesn't create a new file at the given path. It takes the given (existing) image and opens it as tough it wouldn't exist. Pretty much like a template.

**Review by bl-ue — APPROVED** [review]

**Comment by Managor** [comment]
There are ways of rendering images with gimp using its command line. I remember using it to fix some specific thing in an image long ago. If only I remembered what it was, I could contribute it.

**Review by owenvoke — APPROVED** [review]

**Review by einverne — APPROVED** [review]
Nice work!

**Comment by bl-ue** [comment]
>If only I remembered what it was, I could contribute it.

Yeah, that would be nice, but don't feel too much pressure about it.  Plenty of contributors may randomly come a long and help with it 🤷🏻‍♂️

**Pull request merged**